app/cmd/snapshot.go

```diff
@@ -166,6 +168,48 @@ func SnapshotCloneStatusCmd() cli.Command {
 	}
 }
 
+func SnapshotHashCmd() cli.Command {
+	return cli.Command{
+		Name: "hash",
+		Flags: []cli.Flag{
+			cli.StringSliceFlag{
+				Name:  "snapshot-name",
+				Usage: "Specify the name of snapshot needed to be hashed",
+			},
+			cli.BoolFlag{
+				Name:  "rehash",
+				Usage: "Rehash snapshot disk files if necessary",
+			},
+			cli.IntFlag{
+				Name:  "concurrent-limit",
+				Usage: "Concurrent snapshot hashing limit. If set to 0, the setting will not be changed.",
+			},
+		},
+		Action: func(c *cli.Context) {
+			if err := hashSnapshot(c); err != nil {
+				logrus.Fatalf("Error running hash snapshot command: %v", err)
+			}
+		},
+	}
+}
+
+func SnapshotHashStatusCmd() cli.Command {
+	return cli.Command{
+		Name: "hash-status",
+		Flags: []cli.Flag{
+			cli.StringFlag{
+				Name:  "snapshot-name",
```

Comment:
Suggest using a command-line argument instead of as flag.

Reply:
Updated. Thank you.